spec/fixtures/wasm.lua

```diff
@@ -0,0 +1,70 @@
+local _M = {}
+
+local pl_utils = require "pl.utils"
+local pl_dir = require "pl.dir"
+local pl_path = require "pl.path"
+
+local join = pl_path.join
+local execute = pl_utils.executeex
+local abspath = pl_path.abspath
+local basename = pl_path.basename
+local dirs = pl_dir.getdirectories
+local isfile = pl_path.isfile
+local fmt = string.format
+
+_M.ROOT_PATH = "spec/fixtures/proxy_wasm_filters"
+_M.TARGET_PATH = _M.ROOT_PATH .. "/target/wasm32-wasi/debug"
+
+_M.CARGO_BUILD = table.concat({
```

Comment:
I think there's no reason for `CARGO_BUILD` not being `local`. It's just a nitpick, we can address it in a future PR if you agree.